scripts/globals/dynamis.lua

```diff
@@ -113,8 +113,10 @@ local entryInfo =
         beatKI = xi.ki.DYNAMIS_VALKURM_SLIVER,
         enterPos = {100, -8, 131, 47, 39},
         reqs = function(player)
-            return player:hasKeyItem(xi.ki.HYDRA_CORPS_BATTLE_STANDARD) and
```

Comment:
Is it confirmed that having the KI doesn't have an impact on entry?  For example, if they haven't completed Darkness Named, but have it?

Reply:
Am I right in understanding you in that we should try to confirm that clearing all cities and icelands will not guarantee dreamland entry before clearing CoP 3-5? Just making sure I understand you correctly.

Reply:
Yep!  Just want to make sure the Xaracabard clear doesn't short-circuit out of the other requirements, since the KI check was removed.

Reply:
If you want this to be considered first I might push the fix as a stand-alone cause it's an important one.

Reply:
Done. I'll do my best to confirm this as soon as possible or request in captures if not successful.